src/vm/eventpipe.h

```diff
@@ -174,6 +185,9 @@ class EventPipe
     static EventPipeConfiguration s_config;
     static VolatilePtr<EventPipeSession> s_pSessions[MaxNumberOfSessions];
     static EventPipeEventSource *s_pEventSource;
+
+    //! Bitmask tracking EventPipe active sessions.
+    static Volatile<uint64_t> s_activeSessions;
```

Comment:
Let remove the concept of s_activeSession and replace it with s_sessionCount instead, it would make the code much simpler.